src/main/java/org/broadinstitute/consent/http/db/DarCollectionDAO.java

```diff
@@ -50,9 +46,11 @@ public interface DarCollectionDAO {
       DarCollection.FILTER_TERMS_QUERY +
       " )";
 
+  String archiveFilterQuery = " AND ((dar.data #>> '{}')::jsonb->>'status' != 'Archived' OR (dar.data #>> '{}')::jsonb->>'status' IS NULL) ";
```

Comment:
Can we make the clause case insensitive? Here, and throughout:
```suggestion
  String archiveFilterQuery = " AND (LOWER(data->>'status') != 'archived' OR (dar.data #>> '{}')::jsonb->>'status' IS NULL) ";
```